Regression/WarpX-GPU-tests.ini

```diff
@@ -388,7 +388,7 @@ numthreads = 1
 compileTest = 0
 doVis = 0
 compareParticles = 0
-runtime_params = psatd.fftw_plan_measure=0
+runtime_params = psatd.fftw_plan_measure=0 warpx.cfl = 0.5773502691896258
```

Comment:
I'll suggest that it'll be cleaner (and maybe less confusing) to use the default cfl and remove these, and resetting the benchmark cases. Future developers and users will see this and scratch their heads trying to figure out the purpose for the special values of cfl.

Reply:
I would be in favor of doing it in a subsequent PR, just for this, to separate the issues and to have valid regression tests on the changes in this PR (otherwise, these changes could break the code and we'd miss it because we reset the benchmark at the same time).

Reply:
Agreed. In fact, my preference would be to do this separate PR after #1065 is merged.

Reply:
Sounds good.